packages/ui/src/partials/w3m-account-button/styles.ts

```diff
@@ -60,12 +118,28 @@ export function dynamicStyles() {
       color: ${foreground[3]};
     }
 
-    svg path {
-      fill: ${foreground.inverse};
-    }
+    // svg path {
```

Comment:
Will this be un-commented latter or can it be removed? If it is used latter on, could you add TODO here please